src/main/java/thrift/logic/commands/AddExpenseCommand.java

```diff
@@ -15,6 +15,26 @@
 
     public static final String COMMAND_WORD = "add_expense";
 
+    public static final String HELP_MESSAGE = COMMAND_WORD + ": Adds an expense transaction to THRIFT.\n"
+            + "Format: "
+            + COMMAND_WORD + " "
+            + CliSyntax.PREFIX_NAME + "NAME DESCRIPTION "
+            + CliSyntax.PREFIX_VALUE + "VALUE "
+            + "[" + CliSyntax.PREFIX_REMARK + "REMARK] "
+            + "[" + CliSyntax.PREFIX_TAG + "TAG]...\n"
+            + "Possible usages of " + COMMAND_WORD + ": \n"
+            + "To add an expense: "
+            + COMMAND_WORD + " " + CliSyntax.PREFIX_NAME + "Laksa " + CliSyntax.PREFIX_VALUE + "4\n"
+            + "To add an expense with remark: "
+            + COMMAND_WORD + " " + CliSyntax.PREFIX_NAME + "Laksa " + CliSyntax.PREFIX_VALUE + "4 "
+            + CliSyntax.PREFIX_REMARK + "One of my favourite\n"
+            + "To add an expense with tags: "
+            + COMMAND_WORD + " " + CliSyntax.PREFIX_NAME + "Laksa " + CliSyntax.PREFIX_VALUE + "4 "
+            + CliSyntax.PREFIX_TAG + "Bukit Panjang " + CliSyntax.PREFIX_TAG + "Food\n"
```

Comment:
Tags can only consist of singular words - I think you meant `CliSyntax.PREFIX_REMARK`